monitoring/README.md

```diff
@@ -6,112 +6,135 @@
 
-    - `KSController`: select the KubeStellar controller name. Available options from the drop-down menu: *kubestellar-controller-manager-metrics-service*, *ks-transport-controller* and *status-addon-controller*. 
+Note: you can also monitor the APIserver of the hosting cluster using the following values: `NS: default` and `APIService: apiserver`
 
-        <img src="images/ks-controllers-monitoring.png" width="60%" height="80%" title="ks-controllers-monitoring">
+b) API Server Priority & Fairness: input into the following text boxes.
+- `NS`: name of the KubeStellar space namespace, e.g., *wds1-system*, *wds2-system*, *its1-system*, etc.
+- `APIService`: name of the service for a KubeStellar space APIserver, e.g., *wds1*, *wds2*, *vcluster*, etc.
 
-    To monitor a WDS space (e.g., wds1) use the following parameters: 
-    - `SpaceNS: wds1-system` and `KSController: kubestellar-controller-manager-metrics-service`
-    - `SpaceNS: wds1-system` and `KSController: ks-transport-controller`
+    <img src="images/apiserver-apf-monitoring.png" width="60%" height="80%" title="apiserver-apf-monitoring.png"> 
+
+Note: use the same combinations of `NS` and `APIService` parameters described in `step 5-a` to monitor the APIserver in the WDS space, ITS space or the hosting cluster. 
+
+
+c) KubeStellar Controllers Monitoring: input into the following text box and drop-down menu. 
+
+- `SpaceNS`: name of the KubeStellar space namespace, e.g., *wds1-system*, *wds2-system*, etc.
+
+- `KSController`: select the KubeStellar controller name. Available options from the drop-down menu: *kubestellar-controller-manager-metrics-service*, *ks-transport-controller* and *status-addon-controller*. 
+
+    <img src="images/ks-controllers-monitoring.png" width="60%" height="80%" title="ks-controllers-monitoring">
+
+To monitor the KubeStellar controllers in a WDS space (e.g., wds1) use the following parameters: 
+- `SpaceNS: wds1-system` and `KSController: kubestellar-controller-manager-metrics-service`
+- `SpaceNS: wds1-system` and `KSController: ks-transport-controller`
+
+Likewise, to monitor the KubeStellar controller in a ITS space (e.g., its1): `SpaceNS: its1-system` and `KSController: status-addon-controller`
+
+
+#### 6. View Pyroscope profile graphs for KubeStellar controllers in Grafana: 
+
+a) Connect to Pyroscope dashboard: 
+    i) Click `Connections -> Data Sources` in the primary menu
+    ii) Click `Explore` in the Pyroscope datasource selection
+
+b) Query profiles for the KubeSterllar controllers: for each controller, input a profile query (see sample below) and then click the button `Run query` (on the top right).
+- kubestellar-controller-manager: query: `{service_name="wds1-system/manager"}`
+- transport-controller: query: `{service_name="wds1-system/transport-controller"}`
+- addon-status-controller: query: `{service_name="its1-system/status-controller"}`
```

Comment:
This one gets me nothing.
<img width="882" alt="image" src="https://github.com/user-attachments/assets/dde23b30-0e45-44fa-8ef2-283c9bbd2666">

Reply:
I'm aware of this issue - Pyroscope is not scraping the pod in the its1 space although all the configurations are set correctly - I am still looking deep into this and I will create a new PR once I figure out the bug.

Reply:
Fixed! Added a short message to the user.